### ISSUE TITLE
Add "unmaskable" apps to the local run keys

### ISSUE THREAD

**Issue by jhakala** [HCALRunControl/levelOneHCALFM#31]
Certain FMs shouldn't be maskable for certain run types. For example, we should never mask the TCDS FM in most cases, and we should never mask the FM with the hcalLaserController during laser runs. This would also allow us to define run keys for runs with particular partitions -- e.g. it might be more user-friendly to have an "HF only" run key.

**Comment by jhakala** [comment]
I'm bumping this up to medium since there was recently an instance of a user experiencing issues after accidentally masking the TCDS_LPM FM. Also, this functionality is requested for YETS and 904 activities.

**Comment by jhakala** [comment]
This should probably wait until after the grandmaster.

**Comment by sethcooper** [comment]
Were we going to connect this with the single/multipartition functionality in a single run config, right?

**Comment by jhakala** [comment]
I guess maybe that is an idea applicable to 904, but I don't see it being relevant for P5... though we already divide crates up by function manager at 904. What use case do you see?

I was considering closing this with the "won't do" tag since we can always divide things up by FM. What use case do you see being relevant?

**Comment by sethcooper** [comment]
? For YETS when we'll want to have single-partition runs?
I mean, with a multi-partition run, we should prevent the LPM FM from being masked.

**Comment by jhakala** [comment]
Sorry, see my edited last post. I thought you were referring to masking executives (since I didn't read which issue this was before replying).

**Comment by jhakala** [comment]
We should probably consider the cases of unmaskable FMs and unmaskable apps separately now. 

- Unmaskable FMs would be trivial -- simply adding an attribute to the new AVAILABLE_RUN_CONFIGS MapT specifying the list of unmaskable FMs will make this very easy to implement on the GUI side. 
- Unmaskable apps, however, are contingent on first doing #206. Once #206 is done, we can likewise add an attribute to the AVAILABLE_RUN_CONFIGS that specifies the unmaskable apps.

**Comment by jhakala** [comment]
Upping this to high to support the localDAQFM with singlepartition runs.
